rhio/src/blobs/actor.rs

```diff
@@ -124,9 +130,25 @@ impl BlobsActor {
             match event {
                 DownloadBlobEvent::Abort(err) => {
                     error!(parent: &span, %err, "failed downloading blob");
+
+                    metrics::counter!(
```

Comment:
I'd move repeating code in a helper function to improve readability.
`    fn increment_metrics(&self, msg_type: &str, blob: &SignedBlobInfo) {
        metrics::counter!(
            BLOBS_DOWNLOAD_TOTAL,
            LABEL_MSG_TYPE => msg_type.to_owned(),
            LABEL_LOCAL_BUCKET => blob.local_bucket_name.to_owned(),
            LABEL_REMOTE_BUCKET => blob.remote_bucket_name.to_owned()
        ).increment(1);
    }`